.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -363,6 +365,7 @@ kernel_matrix_testing_setup_env:
 kernel_matrix_testing_run_tests_x64:
   extends:
     - .kernel_matrix_testing_run_tests
+  rules: !reference [.except_mergequeue]
```

Comment:
If you intend to skip the tests, then every job in this file can be skipped.

Reply:
I _think_ they should all be skipped with this PR - but I'll double check and see if I can't make it more explicit regardless.

Reply:
Addressed in [0afaafc](https://github.com/DataDog/datadog-agent/pull/20520/commits/0afaafcd3095f0e9ed78334e4438a2d7d0ecc46b)